docs/admin_api/user_admin_api.rst

```diff
@@ -1,3 +1,46 @@
+List Accounts
+=============
+
+This API returns all local user accounts.
+
+The api is::
+
+    GET /_synapse/admin/v2/users?offset=0&limit=10&guests=false
+
+including an ``access_token`` of a server admin.
+The parameters ``offset`` and ``limit`` are required only for pagination.
+Per default a ``limit`` of 100 is used. If the endpoint returns less entries
+than specified by ``limit`` then there are no more users left.
+The parameter ``name`` can be used to filter by user name.
+The parameter ``guests`` can be used to exclude guest users.
```

Comment:
can you document the acceptable values for this? What should I set it to to exclude guest users? Can I use it to return *only* guest users?

Reply:
You can only exclude guests using `guests=false` or `guests=0`.

Reply:
again, can you make this clear in the documentation please. (I'd suggest picking one (`false`?) and documenting that as the correct format, even if you end up supporting `0` too)